core/domain/services/blocks/EventAttendeesBlockRenderer.php

```diff
@@ -53,6 +54,34 @@ public function render(array $attributes)
     }
 
 
+    /**
+     * Get query parameters for model query.
+     *
+     * @param array $attributes
+     * @return array
+     */
+    private function parseGlobalIDs(array $attributes)
+    {
+        // if ticket ID is set, then that's all we need to run the query
+        $ticket = isset($attributes['ticket']) ? $attributes['ticket'] : '';
+        $datetime = isset($attributes['datetime']) ? $attributes['datetime'] : '';
+        $event = isset($attributes['event']) ? $attributes['event'] : '';
+        if ($ticket !== '') {
+            $ticketId = $this->parseGUID($ticket);
+            $attributes['ticketId'] = $ticketId;
+        } elseif ($datetime !== '') {
+            $datetimeId = $this->parseGUID($datetime);
+            $attributes['datetimeId'] = $datetimeId;
+        } elseif ($event !== '') {
```

Comment:
I think it's better to use `if (! empty($ticket))` to take `null` into consideration.

Reply:
that was already handled by the ternaries that are before this block of code.

we first resolved the variables from the array like so:

```php
        $ticket = isset($attributes['ticket']) ? $attributes['ticket'] : '';
        $datetime = isset($attributes['datetime']) ? $attributes['datetime'] : '';
        $event = isset($attributes['event']) ? $attributes['event'] : '';
```

so with php `isset($attributes['ticket'])` will return `false` if that value is `null`
(because `null` is the default value for a variable that has not been set)
which would then set the value of `$attributes['ticket']` to `''`
which can then be quickly and directly compared against using `if ($ticket !== '') {`